justfile

```diff
@@ -331,8 +328,11 @@ linkerd-load: _linkerd-images _k3d-init
         '{{ controller-image }}:{{ linkerd-tag }}' \
         '{{ policy-controller-image }}:{{ linkerd-tag }}' \
         '{{ proxy-image }}:{{ linkerd-tag }}' \
-        "{{ cni-plugin-image }}:$(yq .image.version charts/linkerd2-cni/values.yaml)" \
-        "{{ proxy-init-image }}:$(yq .proxyInit.image.version charts/linkerd-control-plane/values.yaml)" && exit ; sleep 1 ; done
+        $({{ _proxy-init-image-cmd }}) && exit ; sleep 1 ; done
+
+linkerd-load-cni:
```

Comment:
This is a definite improvement but what is supposed to call this rule?

Reply:
When does it need to be called? I left it mostly as a convenience for local testing but none of the tests this is used by require CNI.